[LISKDOCS-30] Add Velodrome (#253)
LiskHQ/lisk-docs: build-blockchain-app.md

To create custom transaction types, utilize the provided `BaseTransaction` interface as described in the section for [custom transactions](custom-transactions.md).

See the [tutorials](tutorials.md) to see examples of how to create custom transaction types and how to add them to your application:

> - [Hello World App](tutorials/hello-world.md): A very basic example of a Lisk blockchain application with 1 custom transaction type.

Reviewer: ```suggestion
- [Hello World App](tutorials/hello-world.md): A example of a Lisk blockchain application with 1 custom transaction type, which extends the BaseTransaction type.
```
I have changed the wording, to make not appear that it is more simple than the Cashback transaction.

> - [Cashback App](tutorials/cashback.md): A simple example blockchain app, that implements the new transaction type `Cashback`.

Reviewer: ```suggestion
- [Cashback App](tutorials/cashback.md): A more practical example creating the new transaction type `Cashback`. Cashback extends the pre-existing transaction type TransferTransaction, making for a straightforward implementation.
```
Re-worded to give some more context to the tutorial
